src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,21 @@
 import React from 'react';
+import './PostList.scss';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
-);
+import { FullPost } from '../../types/FullPost';
+import { PostInfo } from '../PostInfo/PostInfo';
+
+interface Props {
+  posts: FullPost[];
+}
+
+export const PostList: React.FC<Props> = ({ posts = [] }) => {
+  return (
+    <div className="PostList">
```

Comment:
you can return right away here
```suggestion
export const PostList: React.FC<Props> = ({ posts = [] }) => (
    <div className="PostList">
```